### PR TITLE
Updating The Version Of Govmomi For The Installer

### PR DESCRIPTION
**Pull request by ayesha54**
Updating the version of govmomi for the installer and updating the terraform provider for vsphere
https://github.com/openshift/terraform-provider-vsphere/pull/9

### CODE REVIEW

File: data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template [hunk 1]
```diff
@@ -104,10 +104,14 @@ EXTERNAL_IP_OPTIONS="ip=dhcp"
 {{ end }}
 
 
-{{ if .PlatformData.BareMetal.ProvisioningIPv6 }}
-PROVISIONING_IP_OPTIONS="ip=dhcp6"
+{{ if eq .PlatformData.BareMetal.ProvisioningNetwork "Disabled" }}
```
Comment (staebler) [inline data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template:107]: Why are there changes to this file?

File: pkg/asset/ignition/bootstrap/baremetal/template.go [hunk 2]
```diff
@@ -55,6 +55,9 @@ type TemplateData struct {
 
 	// Hosts is the information needed to create the objects in Ironic.
 	Hosts []*baremetal.Host
+
+	// ProvisioningNetwork displays the type of provisioning network being used
+	ProvisioningNetwork string
```
Comment (staebler) [inline pkg/asset/ignition/bootstrap/baremetal/template.go:60]: Likewise, here, these changes should not be in this PR.

File: go.mod [hunk 3]
```diff
@@ -106,276 +106,16 @@ require (
 )
 
 require (
-	cloud.google.com/go/bigtable v1.5.0 // indirect
```
Comment (staebler) [inline go.mod:109]: These lines should not be removed. You may not be running `go mod tidy` properly.

**Review by staebler — COMMENTED** [review]
The PR description claims that this PR updates the "terraform vsphere provider". I don't see that change.

**Comment by ardaguclu** [comment]
Baremetal changes looks unrelated to this PR, probably in next change I'll no more in reviewer list;

/uncc @ardaguclu

File: pkg/asset/ignition/bootstrap/baremetal/template.go (hunk 2 above)
Comment (ayesha54) [reply pkg/asset/ignition/bootstrap/baremetal/template.go]: Thanks @staebler for your comments, actually the PR wasn't completed.

**Comment by patrickdillon** [comment]
/close
govmomi has been bumped in a separate pr

**Pull request closed without merging**